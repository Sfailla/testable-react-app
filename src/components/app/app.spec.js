import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

// TESTING APP
describe('App', () => {
	it('renders App without error', () => {
		render(<App />);
	});

	it('should take a snapshot', () => {
		const { asFragment } = render(<App />);
		expect(asFragment(<App />)).toMatchSnapshot();
	});
});
