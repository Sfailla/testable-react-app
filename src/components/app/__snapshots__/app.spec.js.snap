// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should take a snapshot 1`] = `
<DocumentFragment>
  <header
    class="header-0-2-1"
    data-testid="header"
  >
    <div
      class="wrapper-0-2-3"
    >
      <h2>
        Logo
      </h2>
      <div
        class="navigation-0-2-2"
      >
        <nav
          data-testid="nav"
        >
          <ul
            class="list-0-2-4"
          >
            <li
              class="listItem-0-2-5"
              data-testid="home-link"
            >
              <a
                href="/"
              >
                Home
              </a>
            </li>
            <li
              class="listItem-0-2-5"
              data-testid="form-link"
            >
              <a
                href="/forms"
              >
                Forms
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
  <div
    class="root-0-2-6"
  >
    <h1
      class="heading-0-2-7"
    >
      Welcome To Testable React!
    </h1>
    <div
      class="card-0-2-8 card-d1-0-2-15"
    >
      <div
        class="wrapper-0-2-10"
      >
        <div
          class="banner-0-2-11"
        />
        <img
          alt="react-image"
          class="image-0-2-12"
          src="[object Object]"
        />
        <button
          class="button-0-2-9"
        >
          Open Next Page
        </button>
        <button
          class="button-0-2-9"
        >
          close Window
        </button>
      </div>
    </div>
  </div>
  <footer
    class="footer-0-2-14"
  >
    <p>
      created by steven failla Â© 2020
    </p>
  </footer>
</DocumentFragment>
`;
